packages/mux-audio/package.json

```diff
@@ -13,16 +21,17 @@
   "license": "MIT",
   "scripts": {
     "clean": "shx rm -rf dist/",
-    "test": "web-test-runner **/*test.js --port 8003 --coverage --config test/web-test-runner.config.mjs",
+    "test": "web-test-runner **/*test.js --port 8003 --coverage --config test/web-test-runner.config.mjs --root-dir ../..",
     "posttest": "replace 'SF:src/' 'SF:packages/mux-audio/src/' coverage/lcov.info --silent",
     "dev:iife": "open-process | yarn build:iife --watch",
     "dev:types": "yarn build:types -w",
     "dev": "npm-run-all --parallel dev:types dev:iife",
-    "build:esm": "esbuild src/index.ts --target=es2019 --bundle --sourcemap --format=esm --outdir=dist --out-extension:.js=.mjs --define:PLAYER_VERSION=\"'$npm_package_version'\"",
-    "build:iife": "esbuild src/index.ts --target=es2019 --bundle --sourcemap --format=iife --outdir=dist --define:PLAYER_VERSION=\"'$npm_package_version'\"",
-    "build:cjs": "esbuild src/index.ts --target=es2019 --bundle --sourcemap --format=cjs --outdir=dist --out-extension:.js=.cjs --define:PLAYER_VERSION=\"'$npm_package_version'\"",
+    "build:esm": "esbuild src/index.ts --target=es2019 --bundle --sourcemap --metafile=./dist/esm.json --format=esm --outdir=dist --out-extension:.js=.mjs --external:@mux-elements/* --define:PLAYER_VERSION=\"'$npm_package_version'\"",
+    "build:esm-module": "esbuild src/index.ts --target=es2019 --bundle --sourcemap --metafile=./dist/module.json --format=esm --outfile=./dist/mux-audio.mjs --define:PLAYER_VERSION=\"'$npm_package_version'\"",
```

Comment:
outfile name is `mux-audio.mjs` but `"browser"`/`"unkg"` above are `mux-audio.js`

Reply:
I have the default browser/unpkg build target the iife build.